src/main/java/jalau/at18/katas/game2048/team2/Board.java

```diff
@@ -0,0 +1,234 @@
+package jalau.at18.katas.game2048.team2;
+
+public class Board {
+    private int[][] board;
+    private int score;
+    private int sizeBoard;
+    private final int size = 4;
+    private final int numberThree = 3;
+
+    public Board(int sizeBoard, int score) {
+        this.sizeBoard = sizeBoard;
+        this.score = score;
+        score = 0;
+        board = new int[size][size];
+        createTable();
+    }
+
+    public boolean showTable() {
+        boolean condition = false;
+        for (int indexRows = 0; indexRows < size; indexRows++) {
+            for (int indexColumns = 0; indexColumns < size; indexColumns++) {
+                System.out.print(board[indexRows][indexColumns] + "\t");
+                condition = true;
+            }
+            System.out.println();
+        }
+        return condition;
+    }
+
+    public int[][] createTable() {
+        int[][] matrix = new int[numberThree][numberThree];
+        for (int indexRows = 0; indexRows < size; indexRows++) {
+            for (int indexColumns = 0; indexColumns < size; indexColumns++) {
+                board[indexRows][indexColumns] = 0;
+            }
+        }
+        matrix = board;
```

Comment:
Can `matrix` local variable be removed? I think you don't need because you could assign and modify `board` directly

Reply:
we follow your advice, and remove the matrix variable!!